include/deal.II/grid/grid_tools.h

```diff
@@ -556,6 +556,34 @@ namespace GridTools
   get_active_neighbors (const typename Container::active_cell_iterator        &cell,
                         std::vector<typename Container::active_cell_iterator> &active_neighbors);
 
+  /**
+   * Extract the active cell layer around a subdomain (set of active cells) in
+   * the container @p container (i.e. those that share a common set of vertices)
+   * and return them in the vector @p active_halo_layer.
+   * This subdomain is defined according to the predicate @p predicate. This is
```

Comment:
"This subdomain is defined according to the predicate @p predicate." This is too oblique a definition. How about
```
  Here, the "subdomain" consists of exactly all of those cells for which the @p predicate returns @p true.
```
?